docs/manage/CLI.md

```diff
@@ -29,10 +29,7 @@ viam.component.servo.v1.ServoService.MoveRequest
 
 You can download the Viam CLI executable using one of the options below.
 Select the tab for your platform and architecture.
-
-{{% alert title="Tip" color="tip" %}}
-You can use the `uname -m` command to determine your system architecture.
-{{% /alert %}}
+If you are on Linux, you can use the `uname -m` command to determine your system architecture.
 
 {{< tabs >}}
 {{% tab name="Linux aarch64" %}}
```

Comment:
out of scope for PR, but should macos be first here? (on the assumption it's the most common case)

Reply:
You're right, works for me!